### PR TITLE
Stop the document from jumping to the hash when a token is clicked.

### PR DESCRIPTION
**Pull request by jverkoey**

### CODE REVIEW

**Comment by jverkoey** [comment]
I'm not sure what the best way to stage the integration-specs repo is. It seems that I'd have to land the change in the other repo first in order to reference the commit here. Is that correct?

I'm wondering if instead it's possible to temporarily set the submodule ref to point to my own repo where the relevant integration-specs branch is...

**Comment by pcantrell** [comment]
@jverkoey Yes, you have to first create a commit in `integration-specs` before committing the updated reference in `jazzy`. You also have to make sure that `integration-specs` is pushed before Travis sees your update to `jazzy`.

**Comment by jpsim** [comment]
Ok, I just merged https://github.com/realm/jazzy-integration-specs/pull/7. Could you please update this PR to point the integration specs submodule to that commit?

Change looks good otherwise.

File: CHANGELOG.md [hunk 1]
```diff
@@ -12,7 +12,9 @@
 
 ##### Bug Fixes
 
-* None.
+* Expanding a token no longer causes the document to 'jump' to the hash.
+  [Jeff Verkoeyen](https://github.com/jverkoey)
+  [#352](https://github.com/realm/jazzy/issues/352)
```
Comment (jpsim) [inline CHANGELOG.md]: FYI we only include links to issues, not PRs

Comment (jverkoey) [inline CHANGELOG.md]: Ah good to know - will fix.

**Comment by jpsim** [comment]
Taking this over. Thanks @jverkoey.

**Comment by jverkoey** [comment]
Ah thank you! I got blocked on updating the specs and then life took hold :p is there any way to more easily generate the specs?

**Comment by segiddins** [comment]
There's a rake task that rebuilds them

**Comment by jpsim** [comment]
> is there any way to more easily generate the specs?

unfortunately not. you have push access to `jazzy-integration-specs`, so I suggest you follow these steps when updates are necessary:

```shell
git checkout master
git pull
git checkout -
git rebase master
bundle install
bundle exec rake rebuild_integration_fixtures
cd spec/integration_specs
git checkout -b $jazzy_branch_name
git commit -a -m "update for $jazzy_branch_name"
git push
cd ../../
git commit -a -m "update integration specs"
git push
```

**Comment by jverkoey** [comment]
Noted :) what's the rationale for having the specs as a separate repo?

**Comment by jpsim** [comment]
They're quite large.

**Comment by jverkoey** [comment]
Ah, makes sense. Thanks again!

**Comment by jpsim** [comment]
Actually, they're really not *that* large... I'll make a PR merging them back into jazzy proper so we can discuss the tradeoffs

**Comment by segiddins** [comment]
@jpsim please make the PR from a fork

**Pull request closed without merging**